setup.py

```diff
@@ -0,0 +1,23 @@
+from setuptools import setup, find_packages
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setup(
+    name="graviton",
+    version="0.0.1",
+    url="https://github.com/algorand/graviton",
+    description="verify your TEAL program by experiment and observation",
+    long_description=long_description,
+    author="Algorand",
+    author_email="pypiservice@algorand.com",
+    python_requires=">=3.10",
```

Comment:
Going aggressive here with Python 3.10. Open to other suggestions.

Reply:
@tzaffi I'd like to go to Python 3.10 too.  Presumably, requiring 3.10 here implies the downstream PyTeal PR (e.g. https://github.com/algorand/pyteal/pull/249) must also use 3.10 as a minimum version?

If that's the case, then some work will be needed to bring PyTeal to 3.10 minimum version.  I think the change makes sense.  I'm asking the question to gauge dependencies.

Reply:
This doesn't require anything greater than 3.8. Maybe I'm safe lowering the version here, since all the dependencies getting installed have specific pinned versions. LMK

Reply:
@tzaffi  My primary focus is on bringing the PR to a merge, so I don't want to hold up things.

So, I'm on-board with the change.  But, the following question arises:  Is there a reason _not_ to test the integration tests with the same version matrix?

If there is a rationale, I'm thinking an inline comment to the build is appropriate to benefit future maintainers.

Reply:
After our IRL discussion and further thought, I believe an integration python test matrix is appropriate. Will modify accordingly.